src/components/GraphiQL.js

```diff
@@ -227,6 +232,14 @@ export class GraphiQL extends React.Component {
       this._updateQueryFacts(nextQuery);
     }
 
+    if (!this.props.isKeyHandlerDisabled &&
```

Comment:
sure, still leaning the conventions :) I also added missing semicolons (not sure how significant they are, but as far as i saw, the rest of the code-base uses them)